src/components/AdministratorTable.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <b-container>
+    <b-row class="mt-5 mb-1">
```

Comment:
Margin Top pulls the entire page (including the navbar) down. Would suggest using padding.